### PR TITLE
Biigint ftw wave 2

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

Applies the same logic as https://github.com/dust-tt/dust/pull/9164 but for the connectors database.

All the tables have been migrated to BigInt in production.

## Deploy Plan

No migrations required to be run.

### CODE REVIEW

File: connectors/src/lib/models/confluence.ts [hunk 1]
```diff
@@ -79,15 +62,6 @@ export class ConfluenceSpace extends Model<
 }
 ConfluenceSpace.init(
   {
-    id: {
-      type: DataTypes.INTEGER,
-      autoIncrement: true,
-      primaryKey: true,
-    },
-    connectorId: {
```
Comment (flvndvd) [inline connectors/src/lib/models/confluence.ts:87]: Not needed since we have a relationship defined below.

**Comment by flvndvd** [comment]
PMRR

File: connectors/src/resources/storage/wrappers.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    primaryKey?: boolean;
+    autoIncrement?: boolean;
+  };
+}
+
+/**
+ * A wrapper class that enforces BIGINT for all model primary keys.
+ * All models should extend this class instead of Sequelize's Model.
+ */
+export class BaseModel<M extends Model> extends Model<
```
Comment (JulesBelveze) [inline connectors/src/resources/storage/wrappers.ts:23]: 🔥

**Review by JulesBelveze — COMMENTED** [review]
🔥 💪🏼

**Pull request merged**